nni/retiarii/oneshot/pytorch/sampling.py

```diff
@@ -76,6 +77,17 @@ def training_step(self, batch, batch_idx):
         self.resample()
         return self.model.training_step(batch, batch_idx)
 
+    def export(self) -> dict[str, Any]:
+        """
+        Export of Random one-shot. It will return an arbitrary architecture.
+        """
+        warnings.warn(
+            'Direct export from RandomOneShot returns an arbitrary architecture. '
+            'You might want to use the checkpoint in another search.',
```

Comment:
-> Sampling the best architecture from this well trained supernet is another search process, users can use the supernet's checkpoint to do another search.